live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/transform/response/ATSTransformResponseHandler.java

```diff
@@ -59,41 +50,34 @@ public class ATSTransformResponseHandler extends RouteBuilder
     private static final String ROUTE_ID = "transform-events-consumer";
 
     private final IngestContentCommandHandler ingestContentCommandHandler;
-    private final IngestNodeCommandHandler ingestNodeCommandHandler;
     private final IntegrationProperties integrationProperties;
 
     @Override
     public void configure()
     {
+        // @formatter:off
         SecurityContext securityContext = SecurityContextHolder.getContext();
         from(integrationProperties.alfresco().transform().response().endpoint())
                 .routeId(ROUTE_ID)
                 .log(DEBUG, "Received transform completed event : ${body}")
                 .unmarshal()
                 .json(JsonLibrary.Jackson, TransformResponse.class)
                 .process(exchange -> SecurityContextHolder.setContext(securityContext))
-                .process(this::uploadContentRendition)
-                .process(this::updateContentLocation)
+                .doTry()
+                    .process(this::ingestContent)
+                .doCatch(TransformationFailedException.class)
+                    .log(DEBUG, log, "Transformation failed: ${exception.message}")
```

Comment:
Shouldn't we log a `WARN` or at least `INFO` when transformation will fail? 🤔

Reply:
Currently we expect this to happen for any documents that cannot be converted to pdf. We'll improve the situation when we address https://hyland.atlassian.net/browse/ACS-6488 and then we could log something more useful.